packages/core/src/Box/examples/BoxPolymorphicExample.tsx

```diff
@@ -0,0 +1,33 @@
+import React from 'react';
+import { Box, Flex, TextLink, Button } from '@contentful/f36-components';
+
+export default function BoxSpacingExample() {
+  return (
+    <Flex alignItems="center">
+      <Box as="span" display="inline" marginRight="spacingL">
+        I am span
+      </Box>
+      <Box
+        as="a"
+        display="inline"
+        href="https://contentful.com"
+        target="_blank"
+        marginRight="spacingL"
+      >
+        I am native link
+      </Box>
+      <Box
+        as={TextLink}
```

Comment:
I don't think we should encourage using examples like this one. In the documentation, we should show only examples that we actually recommend, not all the possible options. Why someone should use `Box` to create a `TextLink` instead of using `TextLink`?

Reply:
good point, updated